### DISCUSSION TITLE
No serial output for ESP32-S2 nor ESP32-C3

### DISCUSSION THREAD

**Discussion by rudyv** [HomeSpan/HomeSpan#492] [General]
When deploying to a standard ESP32 everything works, I get serial communication and can key in WiFi redentials.
When I try do do the same for an S2 or C3 I never get the serial communication going.

what am I doing wrong here ?

Rudy

**Comment by HomeSpan** [comment]
Check that you set the baud settings in the serial monitor to what you specified in the sketch.  Also, what sketch are you using for testing - I assume tutorial Example 1?  If you are using your own sketch, make sure you have not used one of the pins on the ESP32 that happen to be associated with the Serial UART on the S2 or C3.

**Reply by rudyv** [reply]
Thanks for the feedback, so with Example 1 it works, It was example 14 I modified myself.
My sincere apologies for the inconvenience
Great project you have, many thanks.

**Reply by HomeSpan** [reply]
Glad to hear it is working.
